include/Layer/Core/ImageNode.hpp

```diff
@@ -14,64 +14,48 @@
  * limitations under the License.
  */
 #pragma once
-#include "Layer/Core/TreeNode.hpp"
+#include "Layer/Core/AttributeAccessor.hpp"
 #include "Layer/Core/PaintNode.hpp"
 #include "Layer/Config.hpp"
-#include "Layer/Core/Transform.hpp"
 #include "Layer/Memory/VNew.hpp"
-#include "Layer/Renderer.hpp"
-#include <core/SkBlendMode.h>
-#include <utility>
-
-class SkImage;
-class SkCanvas;
 
 namespace VGG::layer
 {
 class ImageNode;
-#ifdef USE_SHARED_PTR
```

Comment:
So we are completely removing the fallback support for `std::shared_ptr`? Why?

Nonetheless, `USE_SHARED_PTR` is still recognized in the following files:
```
Layer/ParagraphPainter.hpp
Layer/TreeNode.cpp
Layer/ParagraphLayout.hpp
Layer/SceneBuilder.cpp
Layer/VNode.cpp
```